ui/packages/atlasmap-core/src/contracts/documents/csv.ts

```diff
@@ -66,3 +66,115 @@ export interface ICsvField extends IField {
 export interface ICsvFields {
   csvField: ICsvField[];
 }
+
+export interface IParameterOption {
```

Comment:
oops overlooked at first, can we move these `IParameterOption` and `IParameter` to ../common.ts? They look like generic inspection parameter types that could be used for other document formats.

Reply:
done